.github/workflows/versioneer.yml

```diff
@@ -0,0 +1,32 @@
+# See https://github.com/python-versioneer/python-versioneer
+name: "Update Versioneer"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 1 * *' # 1st day of each month at 06:00 UTC
+  push:
+    paths:
+    - 'setup.cfg'
+    - '.github/workflows/versioneer.yml'
+
+jobs:
+  versioneer:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - run: pip install versioneer black
+    - run: versioneer install
+    - run: |
+        black versioneer.py
+        black geopandas/_version.py
+        black geopandas/__init__.py
+    - uses: actions/upload-artifact@v2
+      with:
+        path: versioneer.py
```

Comment:
Do we need this bit?

Reply:
No necessarily, it's a convenience feature to check the python file if any conflict or other hiccups arise.